components/style/properties/gecko.mako.rs

```diff
@@ -1445,6 +1437,17 @@ fn static_assert() {
 </%self:impl_trait>
 
 
+<%self:impl_trait style_struct_name="InheritedBox"
+                  skip_longhands="image-rendering">
+
+    <% render_keyword = Keyword("image-rendering",
+                                "auto optimizeQuality optimizeSpeed crispEdges") %>
+
+    ${impl_keyword('image_rendering', 'mImageRendering', render_keyword, need_clone=False, style='camel')}
```

Comment:
I would prefer if we just used a snake case enum for this struct, that way further helpers don't have to specialize on the style